<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="Show_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>421</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="addr_label">
       <property name="text">
        <string>监听地址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addr_line"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>139</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="port_label">
       <property name="text">
        <string>监听端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="port_spin">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="maxConnect_label">
       <property name="text">
        <string>最大连接数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxConnect_spin">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="path_label">
       <property name="text">
        <string>主目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="path_line"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="path_Button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>250</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>..</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>195</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="start_Button">
       <property name="text">
        <string>启动</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_Button">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_CurrentTime">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionGuide"/>
   </widget>
   <addaction name="help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_About">
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/picture/images/aboutus.png</normaloff>:/picture/images/aboutus.png</iconset>
   </property>
   <property name="text">
    <string>About QtWebServer</string>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
    </font>
   </property>
  </action>
  <action name="actionGuide">
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/picture/images/guide.ico</normaloff>:/picture/images/guide.ico</iconset>
   </property>
   <property name="text">
    <string>Guide</string>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
